c_api.h

```diff
@@ -72,15 +166,129 @@ void receive_message(unsigned char *received_buf,
 // @param SST_session_ctx_t session ctx struct
 unsigned char *return_decrypted_buf(unsigned char *received_buf,
                                     unsigned int received_buf_length,
+                                    unsigned int *decrypted_buf_length,
                                     SST_session_ctx_t *session_ctx);
 
 // Encrypt the message with session key and send the encrypted message to
 // the socket.
 // @param msg message to send
 // @param msg_length length of message
 // @param SST_session_ctx_t session ctx struct
```

Comment:
Add a comment for return type.

Reply:
Fixed 513882b262c968567e829e51cf96ba6209c6920f.